frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/attached-devices.scss

```
@@ -0,0 +1,11 @@
+.ceph-ocs-install__lso-alert {
+  margin-top: var(--pf-global--spacer--md);
+}
+
+.ceph-create-sc-wizard {
+  overflow-y:auto;
+}
+
+.ceph-ocs-install__no--nodes--text {
```

```suggestion
.ceph-ocs-install__no-nodes-text--large {
```
Check [this](https://seesparkbox.com/foundry/bem_by_example) for more understanding.